<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dAbout</class>
 <widget class="QDialog" name="dAbout">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>513</width>
    <height>619</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>About Ectec</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ectec.qrc">
    <normaloff>:/ectec-icon.svg</normaloff>:/ectec-icon.svg</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="frameGeneral">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>695</width>
            <height>1333</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="spacing">
            <number>24</number>
           </property>
           <property name="leftMargin">
            <number>24</number>
           </property>
           <property name="topMargin">
            <number>24</number>
           </property>
           <property name="rightMargin">
            <number>24</number>
           </property>
           <property name="bottomMargin">
            <number>24</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="labelIcon">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="pixmap">
                <pixmap resource="ectec.qrc">:/ectec-icon.svg</pixmap>
               </property>
               <property name="scaledContents">
                <bool>true</bool>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="labelComponent">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string notr="true">&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Ectec - Component&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;Description of the Component.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
             <property name="openExternalLinks">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelEctec">
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Ectec&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;Ectec is an &lt;span style=&quot; font-weight:600;&quot;&gt;e&lt;/span&gt;ducational &lt;span style=&quot; font-weight:600;&quot;&gt;c&lt;/span&gt;hat &lt;span style=&quot; font-weight:600;&quot;&gt;t&lt;/span&gt;ool for learning how &lt;span style=&quot; font-weight:600;&quot;&gt;e&lt;/span&gt;ncrypted &lt;span style=&quot; font-weight:600;&quot;&gt;c&lt;/span&gt;ommunication works. It is intended to be used in school allowing students to try out encryption methods and protocols they learned as well as possible attack vectors.&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;The source code and more information about the project can be found on its &lt;a href=&quot;https://github.com/real-yfprojects/ectec&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#2980b9;&quot;&gt;Github page&lt;/span&gt;&lt;/a&gt;. Check it out!&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;The &lt;span style=&quot; font-style:italic;&quot;&gt;ectec&lt;/span&gt; programm is written in an easy to learn programming language called &lt;a href=&quot;https://en.wikipedia.org/wiki/Python_(programming_language)&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#2980b9;&quot;&gt;Python&lt;/span&gt;&lt;/a&gt; and its &lt;span style=&quot; font-style:italic;&quot;&gt;C&lt;/span&gt; implementation &lt;a href=&quot;https://www.python.org/&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#2980b9;&quot;&gt;CPython&lt;/span&gt;&lt;/a&gt;. This GUI (graphical user interface) uses the cross-platform &lt;span style=&quot; font-style:italic;&quot;&gt;C++&lt;/span&gt; development framework &lt;a href=&quot;https://doc.qt.io/qt-5/index.html&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#2980b9;&quot;&gt;Qt5&lt;/span&gt;&lt;/a&gt; and its Python bindings &lt;a href=&quot;https://www.riverbankcomputing.com/static/Docs/PyQt5/introduction.html&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#2980b9;&quot;&gt;PyQt5&lt;/span&gt;&lt;/a&gt;. The &lt;span style=&quot; font-style:italic;&quot;&gt;ectec&lt;/span&gt; programm consists of the &lt;span style=&quot; font-style:italic;&quot;&gt;Python&lt;/span&gt; packages `ectec` and `ectecgui`. Both are licensed under the &lt;a href=&quot;https://www.gnu.org/&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#2980b9;&quot;&gt;GNU&lt;/span&gt;&lt;/a&gt; GPL3. The copyright notices and the licensing information can be found in the tab &lt;span style=&quot; font-weight:600;&quot;&gt;License&lt;/span&gt;. The licensing information should also be provided in a file alongside the source code of this program.The GNU GPL3 is an open source and also a free software license (free as in speech opposed to free as in beer). The license is designed with a specific philosophy in mind that is explained in the following.&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;Nobody should be restricted by the software they use. There are four freedoms that every user should have:&lt;/p&gt;&lt;ul style=&quot;margin-top: 0px; margin-bottom: 0px; margin-left: 0px; margin-right: 0px; -qt-list-indent: 1;&quot;&gt;&lt;li style=&quot; margin-top:12px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;the freedom to use the software for any purpose,&lt;/li&gt;&lt;li style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;the freedom to change the software to suit your needs,&lt;/li&gt;&lt;li style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;the freedom to share the software with your friends and neighbors, and&lt;/li&gt;&lt;li style=&quot; margin-top:0px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;the freedom to share the changes you make.&lt;/li&gt;&lt;/ul&gt;&lt;p&gt;When a program offers users all of these freedoms, we call it &lt;a href=&quot;https://www.gnu.org/philosophy/free-sw.html&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#2980b9;&quot;&gt;free software&lt;/span&gt;&lt;/a&gt;.&lt;/p&gt;&lt;p&gt;Developers who write software can release it under the terms of the GNU GPL. When they do, it will be free software and stay free software, no matter who changes or distributes the program. We call this copyleft: the software is copyrighted, but instead of using those rights to restrict users like proprietary software does, we use them to ensure that every user has freedom.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
             <property name="openExternalLinks">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="frameLicense">
      <attribute name="title">
       <string>License</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QScrollArea" name="scrollArea_2">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>493</width>
            <height>532</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLabel" name="licenseView">
             <property name="text">
              <string>License information not available.
A license file should still be located in the application folder.</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
             <property name="indent">
              <number>12</number>
             </property>
             <property name="openExternalLinks">
              <bool>true</bool>
             </property>
             <property name="textInteractionFlags">
              <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="ectec.qrc"/>
 </resources>
 <connections/>
</ui>
